plugins/modules/cloud/lxc/lxc_container.py

```diff
@@ -1700,7 +1693,8 @@ def main():
                 type='str'
             ),
             container_config=dict(
-                type='str'
+                type='list',
+                elements='str'
```

Comment:
Can you also add `type: list` and `elements: str` to the documentation of `container_config`?

Reply:
Done in 773ab76